src/ResourceManager/KeyVault/Commands.KeyVault/Models/KeyAttributes.cs

```diff
@@ -67,12 +65,7 @@ public KeyAttributes()
 
         public DateTime? Updated { get; private set; }
 
-        /// <summary>
-        /// Obsolete flag indicating that the key is protected against purge.
-        /// </summary>
-        /// <remarks>
-        /// Deprecated, will be removed in the next PowerShell release.
-        /// </remarks>
+        [Obsolete("This property is being deprecated and will be removed in a future release")]
```

Comment:
@dragav I would update all warnings with a message like

_"The PurgeDisabled property is being deprecated and will be removed in a future release. Please use the RecoveryLevel property instead."_